datadog.py

```diff
@@ -71,7 +75,8 @@ def get_request_stats(after=None):
             'project_name': app_identity.get_application_id()
         }
         if flavor == 'services' or flavor == 'all':
-            stats['datastore'] = db_stats.GlobalStat.all().get()
+            stats['datastore'] = db_stats.GlobalStat.all().get().to_dict()
```

Comment:
Leo, can you add {{db.to_dict(db_stats.GlobalStat.all().get())}} for db into your branch?